modules/tensor_mechanics/src/base/TensorMechanicsApp.C

```diff
@@ -380,6 +381,7 @@ TensorMechanicsApp::associateSyntax(Syntax & syntax, ActionFactory & action_fact
   syntax.registerActionSyntax("PressureAction", "BCs/Pressure/*");
 
   syntax.registerActionSyntax("GeneralizedPlaneStrainAction", "Modules/TensorMechanics/GeneralizedPlaneStrain/*");
+  syntax.registerActionSyntax("CommonTensorMechanicsAction", "Modules/TensorMechanics/Master");
```

Comment:
Yeah that doesn't work. The Action system needs to know what kind of instance to build when encountering a piece of syntax. Having the Base registered means the Action system would try to build the base type directly.